bots/notify/src/main/java/org/openjdk/skara/bots/notify/JNotifyBot.java

```diff
@@ -170,29 +170,27 @@ private void handleTags(Repository localRepo, UpdateHistory history) throws IOEx
                                 .filter(Optional::isPresent)
                                 .map(Optional::get)
                                 .sorted(Comparator.comparingInt(OpenJDKTag::buildNum))
-                                .map(OpenJDKTag::tag);
+                                .collect(Collectors.toList());
+
         var newNonJdkTags = newTags.stream()
-                                   .filter(tag -> OpenJDKTag.create(tag).isEmpty());
+                                   .filter(tag -> OpenJDKTag.create(tag).isEmpty())
+                                   .collect(Collectors.toList());
```

Comment:
Please move this below the for loop so it is next to the for loop of `newNonJdkTags`

Reply:
Sure!